.github/workflows/python-ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Python CI
+
+on: push
+
+jobs:
+  test:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      env:
+        PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
+      run: |
+        sudo apt-get update
+        sudo apt-get install cdo
```

Comment:
I am just curious why `cdo` was not in our `requirements.txt`?

Reply:
`cdo` isn't a python package! That is why we have to install it separately.